charts/ocis/templates/sharing/config.yaml

```diff
@@ -0,0 +1,15 @@
+{{- include "ocis.basicServiceTemplates" (dict "scope" . "appName" "appNameSharing" "appNameSuffix" "") -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sharing-banned-passwords
```

Comment:
There should be no need to duplicate this ConfigMap.

Just move the one from the frontend service into a "shared" folder and use it for frontend and sharing service

Reply:
which one is is a shared folder?

Reply:
we don't have that yet for Kubernetes resources. We have `_common` which is only used for templates right now. One folder that has no service context is `monitoring`. Therefore we could also just create a `shared` folder or start putting Kubernetes ressources in the `_common` folder :thinking: 

@d7oc what's your opinion on this?

Reply:
I would also vote for shared Kubernetes ressources in this case. I even doubt that it will work at all with two ConfigMaps in the same namespace with the same name. So it has to be shared or the second one has to be renamed.

Reply:
After digging a bit more into it. It's not that easy to have shared resources in the chart. Right now we have a configCheck implemented for the services which require a ConfigMap: https://github.com/owncloud/ocis-charts/blob/0c6101eadeeaa721b8f628bb5ff7a63b034f18e6/charts/ocis/templates/frontend/deployment.yaml#L12 / https://github.com/owncloud/ocis-charts/blob/0c6101eadeeaa721b8f628bb5ff7a63b034f18e6/charts/ocis/templates/_common/_tplvalues.tpl#L203

With shared ConfigMaps this will no longer work and I don't see a quick solution as the template would need to check for the service path and the shared path in the future to use the correct config for build the correct checksum for pod rotation.